docs/ref/models/fields.txt

```diff
@@ -80,22 +80,22 @@ If a field has ``blank=False``, the field will be required.
 
 .. attribute:: Field.choices
 
-A :term:`sequence` consisting itself of iterables of exactly two items (e.g.
+A dictionary consisting of iterables of exactly two items (e.g.
 ``[(A, B), (A, B) ...]``) to use as choices for this field. If choices are
 given, they're enforced by :ref:`model validation <validating-objects>` and the
 default form widget will be a select box with these choices instead of the
 standard text field.
 
-The first element in each tuple is the actual value to be set on the model,
-and the second element is the human-readable name. For example::
+The key element is the actual value to be set on the model, and the second
+element is the human readable name. For example::
 
-    YEAR_IN_SCHOOL_CHOICES = [
-        ('FR', 'Freshman'),
-        ('SO', 'Sophomore'),
-        ('JR', 'Junior'),
-        ('SR', 'Senior'),
-        ('GR', 'Graduate'),
-    ]
+    YEAR_IN_SCHOOL_CHOICES = {
+        'FR': 'Freshman',
+        'SO': 'Sophomore',
+        'JR': 'Junior',
+        'SR': 'Senior',
+        'GR': 'Graduate',
+    }
 
```

Comment:
Add a paragraph here about two-tuple support:

```
You can also pass a list of two-tuples, where the first item is the value and the second is the human readable name, for example::

    YEAR_IN_SCHOOL_CHOICES = (old example)

.. versionchanged: 3.1
    
    Dictionary support was added. Previously Django only supported lists of two-tuples, but since Python 3.6+ dictionaries preserve insertion order so they are just as suitable. 
```

Otherwise people reviewing old code might read the new docs and think two-tuples are entirely unsupported.

Reply:
Thanks! added